### ISSUE TITLE
Location selector on the Event-type form closes on most minimal mouse movement or keyboard event

### ISSUE THREAD

**Issue by raro42** [calcom/cal.com#1600]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

The location selector on the Event-type form (backend) is sensitve to mouse moves and closes the select-pop-up sometimes and sometimes not. In general for a human the form works ... just click again, if the selector closes ... but it is hard to use for a machine (e.g. automated tests).

![2022-01-24 16_06_32-Cometa Meet _ Event-Typ _ Cal com](https://user-images.githubusercontent.com/29251049/150808838-d4cf3316-aaee-4e38-8816-2130094c31e0.png)

### Steps to Reproduce

1. Goto app.cal.com + login
2. Clicked on eventype 
3. See Eventtype details
4. Click on Location 
5. Most minimal mouse movement or hit arrow-key down ... Will close the pop-up with location selection

Workaround:
1. Click on Select location
2. Click somewhere else
3. Click on select again ... now the popup is opened reliable .. mousemovements do not close it anymore

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

It's not really a bug, because a human would just workaround it. But, it def doesn't open reliable. 

Once the selector was opened and closed. Clicking and opening it again is reliable. 

### Technical details

- Browser version: Linux + Windows Chrome 97
- Cometa-Testcase: https://prod.cometa.rocks/#/Default/Stage/457/

**Comment by milospuac** [comment]
Hey Ralf! Thanks for bringing this to our attention. Our team will have a look at this.

**Comment by MarthoxGT** [comment]
Hey @raro42  and @milospuac 

I've been reviewing this problem, and wasn't being able to reproduce it.

If you were using automatic tests, other than the ones already implemented on the main branch, could you please give some more context about the tests you were using please?.

If not, maybe you can upload any kind of multimedia (video) support for us, the readers, to understand better what is happening and how to reproduce it.

Thanks.

**Comment by raro42** [comment]
Cool that you looked into it.

Here is a video explaining it a little bit more.
https://user-images.githubusercontent.com/29251049/151073590-36dd3997-9a64-43c0-be10-d92fa28836b9.mp4

Let me know, if that helped you.

**Comment by raro42** [comment]
And a short video explaining the "arrow-down" provoking the same "closeing of the pop-up". Imho the "pop-up select" is not "reliable" in it's state.

https://user-images.githubusercontent.com/29251049/151074329-ffc9b32f-db04-4871-ad3b-415cb3f127d6.mp4

Could you reproduce that on your side?

**Comment by MarthoxGT** [comment]
@raro42  Thank you very much for the multimedia you provided, it helped a lot to understand better the problem, sadly I wasn't able to reproduce it in (almost) the same conditions, I used a virtual machine with Linux (WSL2 with ubuntu 20.04) a Windows 11 setup and used in both chrome 97.

But I did some research about what could've been happening to see if I can provide any kind of solution approach (theoretically at least), and actually I find something that could help so I will keep an eye on this bug, and I'll report any advance that I think is meaningful to this problem.

Best wishes.
